circleci/docker-publish

```diff
@@ -76,5 +86,6 @@ push_hub_image
 # all repos eventually should dual publish (or explicitly opt-out)
 if [ "$ECR_PUSH_ID" != "" ]; then
   echo "ECR access_id available. Pushing to ECR..."
+  install_awscli
```

Comment:
I think you'll need to install this earlier (e.g. above `ecr_login`) since that also relies on the AWS CLI

Reply:
Like a hawk! Thanks, moved in f2f2d1196e694e985812c0e5b85b406665f11db6